packages/utils/src/type-checks.ts

```diff
@@ -136,6 +136,16 @@ export function isHexadecimal(value: any, prefix = true) {
     return /^[0-9a-fA-F]+$/.test(value)
 }
 
+/**
+ * Checks if the given value can be considered as BigNumber.
+ * A value is considered a BigNumber if it is a bigint or a string
+ * that can be converted to a bigint (via `Bigint(s)`).
+ * @param value The value to check.
+ */
+export function isBigNumber(value: any): boolean {
```

Comment:
Could you also add `big-number` to the `supportedType` list and to the `isType` function (and add tests for it)?

I also wonder if `big-number` and `bigint` could be confusing. What do you think?

Reply:
There's definitely the potential for confusion.  I recently found myself misinterpreting `BigNumber` as being a big version of `number` and worried that it might be a non-integer.

That said, this PR isn't changing the situation.  The types `BigNumber` and `BigNumberish` defined in zk-kit/util introduce that terminology.  I think it's probably not a big issue so long as the usage is internally consistent.

Reply:
Yes, that's something I've seen in `ethers` and some other packages, but I realized it could actually be confusing after looking at the `supportedTypes` list.

Agree it's not related to this PR, which is consistent anyway.